Sources/FTPropertyWrappers/Keychain - Support/KeychainCommons.swift

```diff
@@ -0,0 +1,151 @@
+import Foundation
+
+open class SingleValueKeychainItem {
```

Comment:
Could you please rename this file to `SingleValueKeychainItem.swift`. It only contains this one type.